### DISCUSSION TITLE
Vulnerability contact person or email

### DISCUSSION THREAD

**Discussion by sylwia-budzynska** [dream-num/univer#3493] [Q&A | 常见问题]
Hello univer maintainers,

I've found a few vulnerabilities in dream-num/univer, which I'd like to report. Please, could you let me know how could I share the report with you, e.g. via email or the private vulnerability reporting feature? Thank you!

**Comment by jikkai** [comment]
You can find our public email address (developer@univer.ai) on our official website or GitHub homepage.
